schema/validator.go

```diff
@@ -46,17 +47,17 @@ func (e ValidationError) Error() string {
 }
 
 // Validate validates the given reader against the schema of the wrapped media type.
-func (v Validator) Validate(src io.Reader) error {
+func (v Validator) Validate(src io.Reader, strict bool) error {
```

Comment:
I add doc in https://github.com/opencontainers/image-spec/pull/452/commits/d9b5499553734cfb3e6fcc1588a230ce304560cf#diff-4a672c41642ea30ebf88bb87685cf72aR49. PTAL.